### PR TITLE
Pic-1510: Uppercase/lowercase formatting on matcher journey

### PR DESCRIPTION
**Pull request by sarahyjja**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
     "axe-core": "4.2.3",
     "cypress": "7.6.0",
     "cypress-axe": "0.12.2",
-    "cypress-cucumber-preprocessor": "4.1.2",
+    "cypress-cucumber-preprocessor": "^4.0.0",
```
Comment (theDustRoom) [inline package.json:111]: Don't think you meant to do this?

**Review by theDustRoom — CHANGES_REQUESTED** [review]
This will need to be included everywhere we display the defendant name to ensure we catch them all. Not sure if that's covered by this ticket.

File: package.json (hunk 1 above)
Comment (sarahyjja) [reply package.json]: I think it came when I was doing all these weird manipulation with npm :/

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**